Error in Linux agent installation from sources

## Description

I created a Virtual box machine with Ubuntu 20.04. I then followed these steps from the `Linux - ATP` [documentation](https://documentation-dev.wazuh.com/4.3-rc/deployment-options/wazuh-from-sources/wazuh-agent/index.html):
1. `apt-get install python gcc g++ make libc6-dev curl policycoreutils automake autoconf libtool`
2. `curl -OL https://packages.wazuh.com/utils/cmake/cmake-3.18.3.tar.gz && tar -zxf cmake-3.18.3.tar.gz`
3. `cd cmake-3.18.3 && ./bootstrap --no-system-curl`

Step 3 could not be completed because the following error appeared after a while:
```
[...]
-- Looking for gethostname - found
-- Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY OPENSSL_INCLUDE_DIR) 
CMake Error at Utilities/cmcurl/CMakeLists.txt:485 (message):
  Could not find OpenSSL.  Install an OpenSSL development package or
  configure CMake with -DCMAKE_USE_OPENSSL=OFF to build without OpenSSL.


-- Configuring incomplete, errors occurred!
See also "/home/selu/cmake-3.18.3/CMakeFiles/CMakeOutput.log".
See also "/home/selu/cmake-3.18.3/CMakeFiles/CMakeError.log".
---------------------------------------------
Error when bootstrapping CMake:
Problem while running initial CMake
---------------------------------------------
```

This is what can be seen inside `CMakeError.log`:
```
# tail -n20 /home/selu/cmake-3.18.3/CMakeFiles/CMakeError.log

Determining if the function recv exists in the network;dl failed with the following output:
Change Dir: /home/selu/cmake-3.18.3/CMakeFiles/CMakeTmp

Run Build Command(s):/usr/bin/make cmTC_2557c/fast && make  -f CMakeFiles/cmTC_2557c.dir/build.make CMakeFiles/cmTC_2557c.dir/build
make[1]: Entering directory '/home/selu/cmake-3.18.3/CMakeFiles/CMakeTmp'
Building C object CMakeFiles/cmTC_2557c.dir/CheckFunctionExists.c.o
/usr/bin/gcc   -w -DCHECK_FUNCTION_EXISTS=recv -std=gnu11 -o CMakeFiles/cmTC_2557c.dir/CheckFunctionExists.c.o -c /home/selu/cmake-3.18.3/Modules/CheckFunctionExists.c
Linking C executable cmTC_2557c
/home/selu/cmake-3.18.3/Bootstrap.cmk/cmake -E cmake_link_script CMakeFiles/cmTC_2557c.dir/link.txt --verbose=1
/usr/bin/gcc   -w -DCHECK_FUNCTION_EXISTS=recv CMakeFiles/cmTC_2557c.dir/CheckFunctionExists.c.o -o cmTC_2557c  -lnetwork -ldl 
/usr/bin/ld: cannot find -lnetwork
collect2: error: ld returned 1 exit status
make[1]: *** [CMakeFiles/cmTC_2557c.dir/build.make:106: cmTC_2557c] Error 1
make[1]: Leaving directory '/home/selu/cmake-3.18.3/CMakeFiles/CMakeTmp'
make: *** [Makefile:140: cmTC_2557c/fast] Error 2
```

It seems that OpenSSL is required for the correct installation of CMake. However, it is not listed among the requirements of the first point. It would be necessary to test if the problem is indeed solved by installing OpenSSL and, if so, add it to the list of requirements mentioned in `#1`.

--- a/source/deployment-options/wazuh-from-sources/wazuh-agent/index.rst
+++ b/source/deployment-options/wazuh-from-sources/wazuh-agent/index.rst
@@ -48,8 +48,9 @@ Installing Wazuh agent from sources
 
             .. code-block:: console
 
-              # yum install make gcc gcc-c++ python3 python3-policycoreutils automake autoconf libtool openssl-devel cmake
-              # rpm -i $(rpm -q libstdc++-devel --queryformat "http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/libstdc++-static-%{VERSION}-%{RELEASE}.%{arch}.rpm\n")
+              # yum install make gcc gcc-c++ python3 python3-policycoreutils automake autoconf libtool openssl-devel yum-utils cmake -y
+              # yum-config-manager --enable powertools
+              # yum install libstdc++-static -y
 
             **Optional** CMake 3.18 installation from sources
 
@@ -66,7 +67,7 @@ Installing Wazuh agent from sources
 
         .. code-block:: console
 
-         # apt-get install python gcc g++ make libc6-dev curl policycoreutils automake autoconf libtool
+         # apt-get install python gcc g++ make libc6-dev curl policycoreutils automake autoconf libtool libssl-dev
 
 
         CMake 3.18 installation
@@ -84,7 +85,7 @@ Installing Wazuh agent from sources
 
         .. code-block:: console
 
-         # zypper install make gcc gcc-c++ policycoreutils-python automake autoconf libtool
+         # zypper install make gcc gcc-c++ policycoreutils-python automake autoconf libtool curl libopenssl-devel
 
         CMake 3.18 installation
 

--- a/source/deployment-options/wazuh-from-sources/wazuh-server/index.rst
+++ b/source/deployment-options/wazuh-from-sources/wazuh-server/index.rst
@@ -71,7 +71,7 @@ Installing the Wazuh manager
 
     .. code-block:: console
 
-      # apt-get install python gcc g++ make libc6-dev curl policycoreutils automake autoconf libtool cmake
+      # apt-get install python gcc g++ make libc6-dev curl policycoreutils automake autoconf libtool cmake libssl-dev
 
 
     **Optional** CMake 3.18 installation
@@ -88,7 +88,7 @@ Installing the Wazuh manager
 
     .. code-block:: console
 
-        # zypper install make cmake gcc gcc-c++ policycoreutils-python automake autoconf libtool
+        # zypper install make cmake gcc gcc-c++ policycoreutils-python automake autoconf libtool curl libopenssl-devel
 
     CMake 3.18 installation
 

